demo/mqttv5/mqttv5_client.c

```diff
@@ -207,7 +215,7 @@ static void
 send_callback(nng_mqtt_client *client, nng_msg *msg, void *arg) {
 	nng_aio *        aio    = client->send_aio;
 	uint32_t         count;
-	uint8_t *        code;
+	reason_code *    code;
```

Comment:
Has the error of size of the reason code been fixed?

Reply:
that PR is not yet merged